build/icons.css

```diff
@@ -0,0 +1,160 @@
+}
+
+.icon-bg.iconMarkdown {
+    --bg-icon: url("data:image/svg+xml,%3csvg width='21' height='18' viewBox='0 0 21 18' fill='none' xmlns='http://www.w3.org/2000/svg'%3e %3cpath d='M21 14C21 15.1046 20.1046 16 19 16H2.00098C0.896407 16 0.000976562 15.1046 0.000976562 14V4C0.000976562 2.89543 0.896407 2 2.00098 2L19 2C20.1046 2 21 2.89543 21 4V14ZM4.308 13V8.345L6.5 11.23L8.692 8.345V13H11V5H8.692L6.5 7.885L4.308 5H2V13H4.308ZM19.5 8.99999H17V5H15.0055V8.99999H12.5L16 13.5L19.5 8.99999Z' fill='black'/%3e %3c/svg%3e");
+    width: 21px;
+}
+
+.icon-bg.iconOrderedList {
+    --bg-icon: url("data:image/svg+xml,%3csvg width='18' height='18' viewBox='0 0 18 18' fill='none' xmlns='http://www.w3.org/2000/svg'%3e %3cpath fill-rule='evenodd' clip-rule='evenodd' d='M3 6H4V2H2V3H3V6ZM3.8 8H2V7H5V7.9L3.2 10H5V11H2V10.1L3.8 8ZM2 13V12H5V16H2V15H4V14.5H3V13.5H4V13H2ZM7 5V3H16V5H7ZM7 15H16V13H7V15ZM16 10H7V8H16V10Z' fill='black'/%3e %3c/svg%3e");
+}
+
+.icon-bg.iconPencilSm {
+    --bg-icon: url("data:image/svg+xml,%3csvg width='14' height='14' viewBox='0 0 14 14' fill='none' xmlns='http://www.w3.org/2000/svg'%3e %3cpath d='M11.0937 1.71081L12.2347 2.83019C12.4347 3.03005 12.4347 3.33983 12.2347 3.53968L11.0898 4.70381L9.21 2.86474L10.3837 1.71081C10.5837 1.51095 10.8937 1.51095 11.0937 1.71081Z' fill='black'/%3e %3cpath d='M2 10.12L8.37 3.69427L10.2466 5.57034L3.88 12H2V10.12Z' fill='black'/%3e %3c/svg%3e");
+    width: 14px; height: 14px;
```

Comment:
Can you give me, an actual maniac, a new line between these properties?

Reply:
I could, but changing the ` ` to `\n` in the json gives me this:

```css
.icon-bg.iconPencilSm {
    --bg-icon: url("data:image/svg+xml,...");
    width: 14px;
height: 14px;
}
```

and I'm too lazy to add code to fix it.

> Contributions accepted :trollface: